taxcalc/params.json

```diff
@@ -393,6 +393,19 @@
                     [254200, 	305050,   152525,     279650, 		  305050,  152525],
                     [258250, 	309900,   154950,     284050, 		  309900,  154950]]
     },
+    "_ID_BenefitCap_Switch":{
+        "long_name": "Switch for manipulating itemized deduction benefit cap",
+	"description": "This parameter is specified by items and year. ",
+        "irs_ref": "",
+        "notes": "",
+        "start_year": 2013,
+        "col_var": "MARS",
+        "row_var": "FLPDYR",
+        "row_label": ["2013"],
+        "cpi_inflated": true,
+        "col_label": ["Medical", "StateLocal", "Casualty", "Miscellaneous", "Interest", "Charity"],        
+        "value":    [[1, 	   1,             1,             1,             1,         1]]
```

Comment:
Oh, I see why that might be hard. It would require more logic here:

```
nobenefits_calc.params.ID_casualty_HC = \
nobenefits_calc.params.ID_BenefitCap_Switch[2]
```